src/bananabot.py

```diff
@@ -59,14 +61,20 @@ def start():
             workers[worker.id] = worker
             workers[worker.id].start()
 
+    return True
```

Comment:
Is this boolean necessary? I can't see where its being used

Reply:
See https://github.com/NickoOwen/bananabot/pull/18/files#r1083293567